native/src/components/PoiDetails.tsx

```diff
@@ -136,7 +136,7 @@ const PoiDetails = ({ poi, feature, language }: PoiDetailsProps): ReactElement =
         <>
           <CollapsibleItem initExpanded headerContent={t('description')} language={language}>
             <ContentWrapper>
-              <Page content={content} language={language} path={poi.path} />
+              <Page content={content} language={language} path={poi.path} padding={false} />
```

Comment:
This is new but makes the content consistent with other poi detail row paddings.